ls1intum/Artemis
# `Development`: Fix environment file syntax and name of Docker Compose config file

### Checklist
#### General



### Motivation and Context
One docker compose config environment file didn't follow the [syntax defined by the docker compose specification](https://docs.docker.com/compose/compose-file/05-services/#env_file-format), but it worked, and that's why probably no one has noticed so far. I would now suggest to fix it either way.

Would also rename the file to `prod.env` as there shouldn't be any relation to the `application-local.yml` as config overrides are defined in other files and this file just contains some default docker compose setup configurations for artemis.

### Description
Fixes the syntax of a docker compose config environment file and renames it.

### Steps for Testing
1. Start default docker compose setup
2. Check the loaded profiles defined in `prod.env`

### Review Progress






#### Code Review
#### Manual Tests
